ca/ca.go

```diff
@@ -25,6 +25,7 @@ import (
 const (
 	rootCAPrefix         = "Pebble Root CA "
 	intermediateCAPrefix = "Pebble Intermediate CA "
+	defaultCertLifetime  = 1826
```

Comment:
Let's express this in seconds, rather than days, because certificate validity periods are measured at second-level granularity. Let's also call it `defaultValidityPeriod` (and use "validity period" throughout the rest of the change), as that is the technical name for the thing we're controlling here.

Reply:
Although I agree the granularity of certs is at the second level, I'm not sure what benefit it'll have to also for the cert validity period to have such granularity. But I also have no objection obviously.

Reply:
To me, the two biggest benefits are
1) Then all the math (especially subtracting one second to compute the validity period correctly) is done in the same units; and
2) Boulder's config files measure this in seconds, too.